geth/jail/internal/loop/loop.go

```diff
@@ -101,8 +143,12 @@ func (l *Loop) removeAll() {
 
 // Ready signals to the loop that a task is ready to be finalised. This might
 // block if the "ready channel" in the loop is at capacity.
-func (l *Loop) Ready(t Task) {
+func (l *Loop) Ready(t Task) error {
+	if !l.isAccepting() {
+		return errors.New("The loop is closed and no longer accepting tasks")
```

Comment:
This error message is duplicated, so it's a good candidate of moving into the package scope variable, like:

```go
var ErrClosed = errors.New("The loop is closed and no longer accepting tasks")
...
   return ErrClosed
```